src/greeter/CMakeLists.txt

```diff
@@ -61,16 +61,13 @@ qt_add_lupdate(greeterplugin
     NO_GLOBAL_TARGET
 )
 
-add_dependencies(greeterplugin greeterplugin_lupdate)
```

Comment:
Do not rm directly, Qt 6.7 changed the signature for `qt_add_lrelease`, see https://doc.qt.io/qt-6/qtlinguist-cmake-qt-add-lrelease.html#synopsis
Judge the Qt version and use the different signature for `qt_add_lrelease`